src/lib/drupal/drupalClient.ts

```diff
@@ -13,6 +13,9 @@ export const drupalClient = new DrupalClient(baseUrl, {
     clientId: process.env.DRUPAL_CLIENT_ID,
     clientSecret: process.env.DRUPAL_CLIENT_SECRET,
   },
-  // cache: redisCache(createRedisClient(process.env.REDIS_URL)),
+  // Generally use cache for `yarn export` as it generates all pages
+  cache: process.env.USE_REDIS
```

Comment:
Depending on what we decide with boolean vs. string, this might need to change. `process.env.USE_REDIS` will evaluate to true in the case of strings even if `USE_REDIS=false`. We'd need to check `process.env.USE_REDIS === 'true'`, which I don't particularly love, but it is how things are being done with `SSG` currently (which I think I want to change, as mentioned).